chart/values.yaml

```diff
@@ -85,9 +85,6 @@ base:
   metrics:
     # -- Enable the metrics exporter
     enabled: true
-    # metrics refresh time
-    # WARNING: Lowering pollingInterval value will affect performance adversely
-    pollingInterval: "5m"
```

Comment:
I'm not very clear on how this the exporter works today, but is this configuration now made available somewhere else on the values.yaml?

Reply:
No we dont need this at all. We have removed --polling-interval from metrics exporter container args as well.